web-app/js/records.js

```diff
@@ -36,14 +38,42 @@ var RecordListsViewModel = function(){
     });
   };
 
+  self.delete = function (record) {
+    bootbox.confirm("Are you sure you want to delete the records?", function (result) {
```

Comment:
Shouldn't the message be "Are you sure you want to delete this record?" -> You're only deleting 1.